libs/template/templates/default/template/{{.project_name}}/resources/sample_job.py.tmpl

```diff
@@ -0,0 +1,147 @@
+  {{- end}}
+            },
+{{- end}}
+
+{{- if $python_package}}
+            {
+                "task_key": "python_wheel_task",
+  {{- if $notebook}}
+                "depends_on": [
+                    {"task_key": "notebook_task"},
+                ],
+  {{- end}}
+                "python_wheel_task": {
+                    "package_name": "{{.project_name}}",
+                    "entry_point": "main",
+                    "parameters": [
+                        "--catalog",
+                        "${var.catalog}",
+                        "--schema",
+                        "${var.schema}",
+                    ],
+                },
+  {{- if $serverless}}
+                "environment_key": "default",
+  {{- else}}
+                "job_cluster_key": "job_cluster",
+                "libraries": [
+                    # By default we just include the .whl file generated for the {{.project_name}} package.
+                    # See https://docs.databricks.com/dev-tools/bundles/library-dependencies.html
+                    # for more information on how to add other libraries.
+                    {"whl": "dist/*.whl"},
+                ],
+  {{- end}}
+            },
+{{- end}}
+
+{{- if $pipeline}}
+            {
+                "task_key": "refresh_pipeline",
+  {{- if $notebook}}
+                "depends_on": [
+                    {"task_key": "notebook_task"},
+                ],
+  {{- else if $python_package}}
+                "depends_on": [
+                    {"task_key": "python_wheel_task"},
+                ],
+  {{- end}}
+                "pipeline_task": {
+                    "pipeline_id": "${resources.pipelines.{{.project_name}}_etl.id}",
+                },
+            },
+{{- end}}
+        ],
+{{- if $serverless}}
+        "environments": [
+            {
+                "environment_key": "default",
+                "spec": {
+                    "environment_version": "2",
+  {{- if $python_package}}
+                    "dependencies": [
+                        # By default we just include the .whl file generated for the {{.project_name}} package.
+                        # See https://docs.databricks.com/dev-tools/bundles/library-dependencies.html
+                        # for more information on how to add other libraries.
+                        "dist/*.whl",
+                    ],
+  {{- end }}
+                },
+            },
+        ],
+{{- else }}
+        "job_clusters": [
+            {
+                "job_cluster_key": "job_cluster",
+                "new_cluster": {
+                    "spark_version": "{{template "latest_lts_dbr_version"}}",
+                    "node_type_id": "{{smallest_node_type}}",
+                    "data_security_mode": "SINGLE_USER",
+                    "autoscale": {
+                        "min_workers": 1,
+                        "max_workers": 4,
+                    },
+                },
+            },
+        ],
+{{- end}}
+    }
+)
```

Comment:
I love how familiar this file seems :-) Makes me wonder if there's any additional way to make sure it stays in sync with the YAML version? (non-blocking consideration)